pkg/services/postgresql/all_in_one_bind.go

```diff
@@ -20,18 +20,17 @@ func (a *allInOneManager) Bind(
 	_ service.BindingParameters,
 	_ service.SecureBindingParameters,
 ) (service.BindingDetails, service.SecureBindingDetails, error) {
-	dt, ok := instance.Details.(*allInOnePostgresqlInstanceDetails)
+	dt, ok := instance.Details.(*allInOneInstanceDetails)
```

Comment:
Another pervasive change you will notice: I dropped module/service names from types. The package name already provides this context. It's borderline obnoxious that it was ever done like this to begin with, but it's my own fault, because that is a mistake I made early on in broker development and it's been replicated in other modules as well.